### DISCUSSION TITLE
Editors for dynamic grids

### DISCUSSION THREAD

**Discussion by genyded** [6pac/SlickGrid#673] [Q&A]
Is there an example of or can you provide guidance on using custom Editors in a dynamic/tabbed grid scenarios? The example at https://github.com/6pac/SlickGrid/blob/master/examples/example-dynamic-filtered-with-jquery-tabs.html does not have any editors. However if we follow it and put our editors outside of the CreateGrid function in the example, the 'args' are not set/passed. If we put the editors inside the CreateGrid function, then the references to them in options and elsewhere fails.

Can this example be modified to add at least one custom editor example or is there something elsewhere that shows how best to handle this? We scoured for days and could not find anything.

TIA!

**Comment by ghiscoding** [comment]
That is not an issue with the lib but more of a question so I converted it to a Discussion, please consider that next time. I can't answer your question but this might be better asked on Stack Overflow and provide the code you tried so that someone else can answer your question.

**Reply by genyded** [reply]
We actually believe it is an issue with the grid itself. The editors do not work properly with grids (even a single one) set up using your provided example, specifically the args.item completely disappears and/or the entire row disappears. There are no errors, the row being editing just disappears completely from the grid upon editing or if the edit is aborted and you select another cell, args.item is empty. Stack Overflow has proven useless. I'll see if we can find the time to replicate the issue by adding editors the your provided example as a test case.

**Comment by genyded** [comment]
It turns out the issue was html in an <option> of a select. What is odd is that it worked fine before we moved to dynamic grids, so we're not sure it is was a fluke before or is a timing or some other issue. In any event we were able to resolve it so this can be considered closed.
